service/src/test/java/bio/terra/cbas/dao/TestMethodDao.java

```diff
@@ -0,0 +1,81 @@
+package bio.terra.cbas.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import bio.terra.cbas.models.Method;
+import java.time.OffsetDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+public class TestMethodDao {
+
+  @Autowired MethodDao methodDao;
+  UUID methodId = UUID.fromString("00000000-0000-0000-0000-000000000005");
+
+  @Test
+  void retrievesSingleMethod() {
+    Method testMethod =
+        new Method(
+            methodId,
+            "assemble_refbased",
+            "assemble_refbased",
+            OffsetDateTime.parse("2023-01-27T19:21:24.542692Z"),
+            null,
+            "Github");
+
+    Method actual = methodDao.getMethod(methodId);
```

Comment:
I saw that you added a new `createMethod` function in `MethodDao`. Was the idea to add a method before testing whether it existed or now? That might make this test more resilient to changes in the liquibase state in the future.

Reply:
That was the intention, but this became a bit more complicated for other Dao tests. I decided to keep the `createMethod` for now, since it will be useful in the future when we create more resilient tests.

Reply:
TOL: Would it be better to create new methods/run sets for these DB tests (and at end of test suite remove them from DB) and use them for checks so that these tests don't break if and when we decide to remove the pre-populated workflows from database?

Reply:
This was my initial approach that worked for some Daos, but became complicated with others. I think this is the goal for the second round of db tests to make them more resilient. I'll create a ticket for this work and put it in a placeholder swimlane.